BDArmory/BDArmory.csproj

```diff
@@ -350,6 +350,7 @@
     <Compile Include="Weapons\Missiles\MissileDummy.cs" />
     <Compile Include="Weapons\Missiles\MissileLaunchParams.cs" />
     <Compile Include="Weapons\Missiles\MissileLauncher.cs" />
+    <Compile Include="Weapons\Missiles\ModuleMissileMagazine.cs" />
```

Comment:
This line is getting removed when merging dev into this branch for some reason. (This is more of a note to myself to make sure it's included when merging the PR.)